lib/helper/uploadAgentHelper.py

```diff
@@ -1,13 +1,21 @@
+import os
 import json
 import sys
 import urllib
 import urllib2
 import platform
 import desktopHelper
+import videoHelper
 from datetime import date
+from ..common.pyDriveUtil import PyDriveUtil
 from desktopHelper import DEFAULT_BROWSER_TYPE_FIREFOX
 from desktopHelper import DEFAULT_BROWSER_TYPE_CHROME
 
+DEFAULT_UPLOAD_VIDEO_YAML_SETTING = "./mozhasalvideo.yaml"
+DEFAULT_UPLOAD_VIDEO_MYCRED_TXT = "./mycreds_mozhasalvideo.txt"
```

Comment:
We just need to put those two files in working dir, and those two credential are generated based on our test google account